modal/image.py

```diff
@@ -197,7 +197,7 @@ async def _load(resolver: Resolver, existing_object_id: Optional[str], handle: _
                 build_function_def = build_function.get_build_def()
                 build_function_id = (await resolver.load(build_function)).object_id
 
-                globals = build_function.get_globals()
+                globals = build_function._get_info().get_globals()
```

Comment:
Minor thing (because this is some cruft I introduced) but we should probably replace `_get_info` with a property instead. In general let's avoid accessing internals of other objects.

But I can fix separately since it's mostly my fault that the `Function` interface is a bit janky